packages/FlexGrid/Row/Row.md

```diff
@@ -4,21 +4,50 @@ The `FlexGrid.Row` component can be used for Column alignment and distribution.
 
 ```jsx
 <FlexGrid>
+  <FlexGrid.Row horizontalAlign="start">
+    <FlexGrid.Col xs={2}>Left</FlexGrid.Col>
+    <FlexGrid.Col xs={2}>aligned</FlexGrid.Col>
+  </FlexGrid.Row>
   <FlexGrid.Row horizontalAlign="center">
-    <FlexGrid.Col span={3}>Have a nice day</FlexGrid.Col>
-    <FlexGrid.Col span={3}>You seem healthy</FlexGrid.Col>
-    <FlexGrid.Col span={3}>Take care</FlexGrid.Col>
+    <FlexGrid.Col xs={2}>Center</FlexGrid.Col>
+    <FlexGrid.Col xs={2}>aligned</FlexGrid.Col>
+  </FlexGrid.Row>
+  <FlexGrid.Row horizontalAlign="end">
+    <FlexGrid.Col xs={2}>Right</FlexGrid.Col>
+    <FlexGrid.Col xs={2}>aligned</FlexGrid.Col>
   </FlexGrid.Row>
 </FlexGrid>
 ```
 
 ```jsx
 <FlexGrid>
-  <FlexGrid.Row horizontalAlign="center" verticalAlign="bottom">
-    <FlexGrid.Col span={3}>Hello, have a nice day!</FlexGrid.Col>
-    <FlexGrid.Col span={3}>You seem healthy</FlexGrid.Col>
-    <FlexGrid.Col span={3}>Take care</FlexGrid.Col>
+  <FlexGrid.Row horizontalAlign="center" verticalAlign="top">
+    <FlexGrid.Col xs={5}>
+      <Input label="Given name" value="John" />
+    </FlexGrid.Col>
+    <FlexGrid.Col xs={5}>
+      <Input label="Surname" value="Appleseed" />
+    </FlexGrid.Col>
   </FlexGrid.Row>
+
+  <Box tag="section" inset={2}>
+    <FlexGrid.Row horizontalAlign="center" verticalAlign="bottom">
+      <FlexGrid.Col xs={5} md={4} lg={3}>
+        <Checkbox name="test" value="terms" label="I accept the ToS" />
+      </FlexGrid.Col>
+      <FlexGrid.Col xs={5} md={4} lg={3}>
+        <Checkbox name="services" value="emails" label="I accept cookies" />
+      </FlexGrid.Col>
+    </FlexGrid.Row>
+  </Box>
+
+  <Box inset={3}>
+    <FlexGrid.Row horizontalAlign="center" verticalAlign="bottom">
+      <FlexGrid.Col xs={12}>
+        <Button>Submit</Button>
+      </FlexGrid.Col>
+    </FlexGrid.Row>
+  </Box>
```

Comment:
Honestly I think this is getting a bit overly creative. :) I think the extra components (`Input`, `Button`, `Checkbox`) will obscure the purpose of these examples, which is to give readers and understanding of the grid props and behaviour.

I think using simple language or just solid color in the grid content is superior, as we have done in previous examples. Check out some examples of well documented grids here:

* https://getbootstrap.com/docs/4.0/layout/grid/
* https://semantic-ui.com/collections/grid.html
* https://roylee0704.github.io/react-flexbox-grid/